types/react-big-calendar/index.d.ts

```diff
@@ -133,6 +142,41 @@ export interface Components {
     };
 }
 
+export interface ToolbarProps {
+    date: Date;
+    view: View;
+    views: Views;
+    label: string;
+    localizer: { messages: Messages };
+    onNavigate: (navigate: Navigate, date?: Date) => void;
+    onView: (view: View) => void;
+    children?: React.ReactNode;
+}
+
+export interface EventWrapperProps<T extends Event = Event> {
+    style?: React.CSSProperties & { xOffset: number };
```

Comment:
Just leaving this here https://github.com/intljusticemission/react-big-calendar/blob/27a2656b40ac8729634d24376dff8ea781a66d50/src/TimeGridEvent.js#L28